### ISSUE TITLE
Mat-Select with long options text blinks in IE

### ISSUE THREAD

**Issue by RandomNonameDude** [angular/components#11642]
#### Bug, feature request, or proposal:

When there are options with long text in the options list in `mat-select`, the options are not shown whole when the list is open - which is, of course, correct. However, in Internet Explorer, the options somehow blinks to their full length and back to the truncated version when one opens or closes the list or when one selects an option.

#### What is the expected behavior?

The options list should not blink anyhow in Internet Explorer.

#### What is the current behavior?

When there is long text in select's options list, the list blinks when it is opened, closed, or when an option is selected.

#### What are the steps to reproduce?

https://stackblitz.com/edit/angular-qdcxkp?file=app/select-overview-example.ts

Open in Internet Explorer and:
- Click the mat-select (first blink) and then click away (second blink). 
- Click the mat-select (first blink) and then select an option (second blink). 

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

Win7, IE.

**Comment by macjohnny** [comment]
this seems somehow related to https://github.com/angular/material2/issues/11609

**Comment by macjohnny** [comment]
This issue would be partly fixed with https://github.com/angular/material2/pull/11801, however there still is a small jump due to an issue with animating `calc()` in IE11: https://github.com/angular/angular/issues/24545

see https://github.com/angular/material2/pull/11801#issuecomment-397805702 for a detailed analysis of the problem

**Comment by macjohnny** [comment]
@crisbeto please let me know whether you would accept a fix that removes the animation from `100%` to `100% + 32px`, so I could file another PR for this.

**Comment by Bamyan** [comment]
browser **Edge 17.17134.0** on platform **Windows 10** is also affected

**Comment by MaiLeeTR** [comment]
This is still an issue with shorter options as well. I saw that this [issue](https://github.com/angular/components/issues/11609) was closed and fixes were merged, but I don't think it's fixed. 

If you go to the mat-select examples and look at the [basic select example](https://material.angular.io/components/select/overview), the options are not long and the width of the panel still flickers when opening. 

Should I submit a new issue for this or will this be solved with this current issue?
